### PR TITLE
Added marker logic for payable leases

### PR DESCRIPTION
**Pull request by FuriousLlama**
image

### CODE REVIEW

File: source/frontend/src/components/maps/leaflet/mapUtils.tsx [hunk 1]
```diff
@@ -114,13 +137,22 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
...
  */
 export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
+  if (feature.properties.IS_PAYABLE_LEASE || feature.properties.isPayableLease) {
+    if (selected) {
+      return propertyWithLeaseIconSelect;
+    } else {
+      return propertyWithLeaseIcon;
+    }
+  }
+
```
Comment (asanchezr) [inline source/frontend/src/components/maps/leaflet/mapUtils.tsx:147]: Did you update the marker legend to include this new pin? I mean this map component...
image

Comment (FuriousLlama) [reply source/frontend/src/components/maps/leaflet/mapUtils.tsx:147]: uh nope. Never pressed that button before. Guess thats what they meant by 'label' :)

File: source/frontend/src/components/maps/leaflet/Legend/Legend.tsx [hunk 2]
```diff
@@ -17,6 +17,10 @@ export const Legend = () => {
         pin: require('assets/images/pins/land-poi.svg').default,
         label: 'Property of Interest',
       },
+      {
+        pin: require('assets/images/pins/land-lease.svg').default,
+        label: 'Payable lease/license',
+      },
```
Comment (asanchezr) [inline source/frontend/src/components/maps/leaflet/Legend/Legend.tsx:23]: not a big deal but I see four (4) images added on this PR - 2 svgs and 2 pngs but I only see the svgs being used in code. Are the pngs used or could they be safely removed?

**Review by asanchezr — APPROVED** [review]

File: source/frontend/src/components/maps/leaflet/Legend/Legend.tsx (hunk 2 above)
Comment (FuriousLlama) [reply source/frontend/src/components/maps/leaflet/Legend/Legend.tsx:23]: My guess is that they could be removed if not used. But I did not check if the other marker images have the same usage.

**Pull request merged**